sotodlib/sim_hardware.py

```diff
@@ -573,8 +582,9 @@ def sim_wafer_detectors(hw, wafer_slot, platescale, fwhm, band=None,
                 dprops["channel"] = doff
                 dprops["AMC"] = doff // chan_per_AMC
                 dprops["bias"] = doff // chan_per_bias
-                dprops["reset_rate_kHz"] = 4.
                 dprops["readout_freq_GHz"] = readout_freq[doff]
+                dprops["bondpad"] = doff-(doff//64)*64
```

Comment:
I would like to see the numerical constant `64` assigned to some descriptive variable name rather than used bare. Perhaps declared next to `chan_per_AMC`?

Reply:
Good call--I added a variable for it